### PR TITLE
Legion can now salvage Veteran Ranger and Power Armour

### PR DESCRIPTION
**Pull request by BrotherHangyul**
## About The Pull Request

Ports PR #69 from the Lonestar Github, as well as other PR's that improved the sprites or otherwise balanced out the armours. Legion players now start with the crafting recipes for, what is essentially, nerfed ranger-hunter and power-armour slayer armour. 

Essentially; players can convert Veteran Ranger armour and Salvaged PA, both normal and NCR, into slightly worse statistically but better looking pieces of armour.

## Why It's Good For The Game

Drip.

Also, alternate armour that is actually, properly modified to be Legion is both canon to New Vegas lore (people regularly take parts off of other people's armours to make good ones) and is just fun to do. Furthermore, it rewards killing (arguably) hard-to-kill targets like the Heavy Trooper and the Veteran Ranger with armour that, whilst mostly worse, is pretty.

Also also, Titus Aurelius asked, and I'm not gonna deny him that like cmon he'd find out where I live and stab me.

Everything's tested and has been error checked like 50 times.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Added crafting recipes and the armours of salvaged veteran ranger and power armour, legion-style.
imageadd: Added sprites for the salvaged armours

/:cl:

### CODE REVIEW

**Comment by BrotherHangyul** [comment]
image
image
How this shit looks for reference.

**Comment by ZAB909** [comment]
extreme drip 1+

**Comment by BadAtThisGame302** [comment]
Make it only for NCR variants of salvaged armor, not all of them.

**Comment by BrotherHangyul** [comment]
> Make it only for NCR variants of salvaged armor, not all of them.

It's only the NCR veteran ranger armour for the Legion Riot Gear, the heavy legion armour is compatible with both NCR salvaged power armour and normal salvaged power armour as that just seems to be logical.

**Comment by BadAtThisGame302** [comment]
> > Make it only for NCR variants of salvaged armor, not all of them.
> 
> It's only the NCR veteran ranger armour for the Legion Riot Gear, the heavy legion armour is compatible with both NCR salvaged power armour and normal salvaged power armour as that just seems to be logical.

It's logical but is it balanced? No.

**Comment by ZAB909** [comment]
If they find PA they're gonna make it based and bull pilled, so w/e

**Comment by BrotherHangyul** [comment]
> > > Make it only for NCR variants of salvaged armor, not all of them.
> > 
> > 
> > It's only the NCR veteran ranger armour for the Legion Riot Gear, the heavy legion armour is compatible with both NCR salvaged power armour and normal salvaged power armour as that just seems to be logical.
> 
> It's logical but is it balanced? No.

Legion heavy armour has nearly identical stats to normal salvaged power armour and is arguably worse. It's literally a cosmetic difference.

**Comment by chemistrymain2** [comment]
> > > Make it only for NCR variants of salvaged armor, not all of them.
> > 
> > 
> > It's only the NCR veteran ranger armour for the Legion Riot Gear, the heavy legion armour is compatible with both NCR salvaged power armour and normal salvaged power armour as that just seems to be logical.
> 
> It's logical but is it balanced? No.

It's just spraypainted SPA. It's fine, since Legion sucks if slow.

File: code/modules/clothing/head/f13factionhead.dm [hunk 1]
```diff
@@ -471,6 +471,23 @@ obj/item/clothing/head/helmet/f13/enclave/usmcriot
...
+	armor = list("melee" = 40, "bullet" = 50, "laser" = 40, "energy" = 15, "bomb" = 55, "bio" = 60, "rad" = 60, "fire" = 90, "acid" = 20, "wound" = 55)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR|HIDEFACE
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	resistance_flags = LAVA_PROOF | FIRE_PROOF
+	
+/obj/item/clothing/head/helmet/f13/legion/legionsalvagedhelmet
+	name = "legion heavy helmet"
+	desc = "A once-operational power armor helmet, repurposed by the Legion into a symbol of its might, sporting a bronze visor."
+	icon_state = "legion-heavy"
+	item_state = "legion-heavy"
```
Comment (BadAtThisGame302) [inline code/modules/clothing/head/f13factionhead.dm:489]: You forgot 0.1 slowdown on the helmet.

Comment (BrotherHangyul) [reply code/modules/clothing/head/f13factionhead.dm:489]: I didn't. It's intended.

**Pull request merged**